build.sbt

```diff
@@ -66,6 +66,18 @@ lazy val root = Project(id = "root", base = file("."))
     )
   )
 
+lazy val macros = Project(id = "content-api-models-macros", base = file("macros"))
+  .settings(commonSettings)
+  .settings(
+    description := "Macros",
+    libraryDependencies ++= Seq(
+      "io.circe" %% "circe-core" % "0.5.0-M2",
```

Comment:
Maybe worth extracting the Circe version into a `val` so we can keep it consistent between the projects